corehq/apps/app_manager/models.py

```diff
@@ -2713,12 +2713,27 @@ def validate_for_build(self):
         return errors
 
 
+class ReportGraphConfig(DocumentSchema):
```

Comment:
Hey, can you say a bit about how/why ReportGraphConfig and GraphConfiguration are different classes? Their properties look similar, though not the same.